src/simulators/statevector/qubitvector.hpp

```diff
@@ -363,6 +366,19 @@ class QubitVector {
   template <typename Lambda>
   void apply_lambda(Lambda&& func);
 
+  //-----------------------------------------------------------------------
+  // Statevector update with Lambda function on a range of entries
+  //-----------------------------------------------------------------------
+  // Apply a lambda function to all entries of the statevector
+  // between start and stop
+  // The function signature should be:
+  //
+  // [&](const int_t k)->void
+  //
+  // where k is the index of the vector
+  template <typename Lambda>
+  void apply_lambda(Lambda&& func, size_t start, size_t stop);
```

Comment:
Why not just call `QV::apply_lambda(start, stop, omp_threads_managed(), func)` directly instead of adding this function.

Reply:
Have to admit I'm not sure why it mattered to me... I'll fix as you suggested.

Reply:
Purely cosmetic, but in line with the other `apply_lambda` functions defined in this file.